docs/helper-functions.md

```diff
@@ -0,0 +1,64 @@
+## Helper Functions
+
+**`whitelist_html`**
+
+When translating strings in WordPress, the most common functions to use are
+`__()` (translate and return) or `_e()` (translate and output). Where possible,
+these need to be escaped too, to ensure that translations don't accidentally
+break your output. For this reason, `esc_html_e()`, `esc_attr_e()`, etc are
+offered by WordPress as convenience functions.
+
+However, this falls down when you need to have HTML in the translation.
+
+This function provides a nice, easy, performant way to perform sanitization on translated strings. Rather than requiring you to work with the internals of kses, it's much closer to functions like `esc_html`.
```

Comment:
```suggestion
This function provides a nice, easy, performant way to perform sanitization on translated strings. Rather than requiring you to work with the internals of kses, it's much closer to functions like `esc_html()`.
```
When referring to functions lets keep this consistent throughout the document - it's clearer that it's a function if we include the round brackets